content/cloud/noops-sql-db.md

```diff
@@ -0,0 +1,121 @@
+
+## Accessing Private Beta
+
+NoOps SQL Database in Fermyon Cloud is currently in private beta. To request access to the private beta, please fill out this short [sign-up form](https://fibsu0jcu2g.typeform.com/to/Brv12FI0#hubspot_utk=xxxxx&hubspot_page_name=xxxxx&hubspot_page_url=xxxxx).
+ 
+> Please note that the private beta is limited in space and all requests cannot be guaranteed. 
+
+## Service Limitations and Quotas For NoOps SQL Database in Fermyon Cloud
+
+*Service Limitations*
+* You must delete your Spin application before you can delete your NoOps SQL Database
+* The `--sqlite` flag not supported on `spin cloud deploy`
+* Fermyon Cloud only supports "default" NoOps SQL Databases at this time
+
+*Quotas* 
+* You can have a maximum of 1 NoOps SQL Database in your Cloud account
+* Your NoOps SQL Database can hold up 1 GB of data
+
+## How To:
+
+### Prepare a NoOps SQL Database
+
+We recommend using the `execute` command in your application's source code to create a table. For example, in this routing function we use the Rust SDK to check if the table exists. If it doesn't, then we create one:
+
+```rust
+#[http_component]
+fn handle_todo(req: Request) -> anyhow::Result<Response> {
+    let connection = Connection::open_default()?;
+		connection.execute("CREATE TABLE IF NOT EXISTS todos (
+			  id INTEGER PRIMARY KEY AUTOINCREMENT,
+			  description TEXT NOT NULL,
+			  due_date DATE,
+			  starred BOOLEAN DEFAULT 0,
+			  is_completed BOOLEAN DEFAULT 0
+				);", &[])?;
+
+    let router = http_router! {
+<...>
+    };
+    router.handle(req)
+}
+```
+
+### Manage NoOps SQL Database Lifecycle
+
+A NoOps SQL Database is created when you deploy a Spin application that references a "default" SQLite database in it's component manifest. Multiple components in the same Spin application can be given access to the default database like so:
+
+```toml
+# c1 does not have access to the default database
+[component]
+name = "c1"
+
+# c2 can use the default database
+[component]
+name = "c2"
+sqlite_databases = ["default"]
+
+# c3 can use default database as well
+[component]
+name = "c3"
+sqlite_databases = ["default"]
+```
+
+The NoOps SQL database will created when the Spin application is deployed. The data in between Spin application invocations and updates. 
+
+```bash
+$ spin cloud deploy
+Deploying app ..
+Creating database inspirational-pig (default) for app "quickstart"
+```
+
+You can check to see which Spin application is tied to your NoOpsSQL database by using the `spin cloud sql` command:
+
+>> Fermyon Cloud will automatically generate a random name for your NoOps SQL database.
+
+```bash
+$ spin cloud sqlite list
+inspirational-pig (default) (currenly used by "quickstart")
+```
+
+Lastly, after [deleting your Spin application](/cloud/delete.md) you can delete your default database with the following command:
+
+```bash
+$ spin cloud sqlite delete default
+inspirational-pig has been deleted
```

Comment:
I am not sure what output we have on deletes currently. this may need to be updated... or the cli updated to match this

Reply:
The output has been updated, so this should be updated.